packages/ragbits-core/tests/unit/llms/factory/test_get_default_llm.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
```

Comment:
I think it would be worth adding couple of tests - to check that getting different types of LLMs work (and not only the default text type) and that using an nonexistent type in pyproject generates an exception

Reply:
Tests added